src/plasma/plasma_manager.cc

```diff
@@ -1311,33 +1312,35 @@ void log_object_hash_mismatch_error_object_callback(ObjectID object_id,
 }
 
 void process_add_object_notification(PlasmaManagerState *state,
-                                     ObjectInfo object_info) {
-  ObjectID obj_id = object_info.obj_id;
+                                     ObjectID object_id,
```

Comment:
It is to make the code independent of the concrete ObjectInfoT representation (the main point being that ObjectInfoT uses std::string to represent object_ids and this method should not be aware of that fact).